draft-song-dns-wireformat-http-01.txt

```diff
@@ -271,17 +274,17 @@ Internet-Draft          DNS wire-format over HTTP          February 2016
 
    Proxy-DNS-Transport: xyz
```

Comment:
should we make this header field optional or mandatory? if mandatory we should ask server always to check and follow this field. if not optional, then the server should have a default behavior if this field missing.